Add convenient function obj_fetch/3 which returns default-value

fnchooft opened the pull request:
The option for obj_fetch is either an atom or a runtime error.
However one could return a default-value - simplifying code.

vinoski commented:
Thanks! I expanded your commit a bit to add a new unit test, and when doing so I found that a couple other `json2` unit tests weren't doing anything at all because they were looking in the wrong place for their test files, so I fixed that too. I've rebased this modified commit to master.

fnchooft commented:
Thank you Steve!

Note to self: Run / Fix unit-tests for future development.

Kind regards,

Fabian

On Wed, Apr 7, 2021 at 10:18 AM Steve Vinoski ***@***.***>
wrote:

> Thanks! I expanded your commit a bit to add a new unit test, and when
> doing so I found that a couple other json2 unit tests weren't doing
> anything at all because they were looking in the wrong place for their test
> files, so I fixed that too. I've rebased this modified commit to master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/erlyaws/yaws/pull/421#issuecomment-814907530>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABD6LCJ3GO6HB7RACKGP3TDTHRLSRANCNFSM42P5XHUQ>
> .
>
